inc/core/styles/css_prop.php

```diff
@@ -309,11 +309,11 @@ public static function transform_directional_prop( $meta, $device, $value, $css_
 		}, ARRAY_FILTER_USE_KEY );
 
 		if ( count( array_unique( $filtered ) ) === 1 ) {
-			if ( abs( $value['top'] ) === 0 ) {
+			if ( neve_value_is_zero( $value['top'] ) ) {
 				$suffix = '';
 			}
 
-			if ( empty( $value['top'] ) && abs( $value['top'] ) !== 0 ) {
+			if ( empty( $value['top'] ) && ! neve_value_is_zero( $value['top'] ) ) {
```

Comment:
you might not need this check as if the value is `empty()` `neve_value_is_zero()` will return `true`